src/openrct2/actions/ParkSetLoanAction.cpp

```diff
@@ -42,21 +42,19 @@ GameActions::Result ParkSetLoanAction::Query() const
 {
     auto currentLoan = gBankLoan;
     auto loanDifference = currentLoan - _value;
-    if (_value > currentLoan)
+    if (_value > currentLoan && _value > gMaxBankLoan)
     {
-        if (_value > gMaxBankLoan)
-        {
-            return GameActions::Result(
-                GameActions::Status::Disallowed, STR_CANT_BORROW_ANY_MORE_MONEY, STR_BANK_REFUSES_TO_INCREASE_LOAN);
-        }
+        return GameActions::Result(
+            GameActions::Status::Disallowed, STR_CANT_BORROW_ANY_MORE_MONEY, STR_BANK_REFUSES_TO_INCREASE_LOAN);
     }
-    else
+    if (_value < currentLoan && _value < 0)
```

Comment:
```suggestion
    if (_value < currentLoan && _value < 0.00_GBP)
```

Reply:
Is that better? `_value` is a `money64`, but `0.00_GBP` is a `money32`. Afaik there is no `money64` literal, but I could easily add it if you think it makes sense.

Reply:
I think we should be using `money64` in as many places as possible, really, and change the _GBP literal to it as well. Until that time, I’d put a TODO or FIXME comment here and not use the _GBP literal yet.